app/views/ops/_settings_rhn_edit_tab.html.haml

```diff
@@ -92,10 +92,10 @@
           = _("Organization")
         .col-md-8
           - if @edit[:organizations].length == 1
-            = @edit[:new][:customer_org]
+            = @edit[:new][:customer_org_display]
           - else
             = select_tag('customer_org',
-                          options_for_select((@edit[:new][:customer_org] ? [] : [["<Choose>", nil]]) + Array(@edit[:organizations].sort_by { |k, _v| k.downcase }),
+                          options_for_select((@edit[:new][:customer_org_display] ? [] : [["<Choose>", nil]]) + Array(@edit[:organizations].sort_by { |k, _v| k.downcase }),
```

Comment:
@carbonin Well, I ask because it's being used here, so if it never gets set, I don't see how this part can work.